package.json

```diff
@@ -32,7 +32,6 @@
     "lint:fix": "prettier --write ./**/*.json && eslint . --ext js --fix",
     "lint:changed": "{ git ls-files --others --exclude-standard ; git diff-index --name-only --diff-filter=d HEAD ; } | grep --regexp='[.]js$' | tr '\\n' '\\0' | xargs -0 eslint",
     "lint:changed:fix": "{ git ls-files --others --exclude-standard ; git diff-index --name-only --diff-filter=d HEAD ; } | grep --regexp='[.]js$' | tr '\\n' '\\0' | xargs -0 eslint --fix",
-    "lint:shellcheck": "./development/shellcheck.sh",
```

Comment:
Should we keep this for running `shellcheck` locally?

Reply:
Sure, I've reverted this line